utils/project/sync.go

```diff
@@ -143,3 +154,56 @@ func completeUpload(projectId string, files []string, modfiles []string, timesta
 		// TODO - Need to handle this gracefully.
 	}
 }
+
+func ignoreFileOrDirectory(name string, isDir bool) bool {
+	// List of files not add to PFE
+	ignoredFiles := []string{
+		".DS_Store",
+		"*.swp",
+		"*.swx",
+		"Jenkinsfile",
+		".cfignore",
+		"localm2cache.zip",
+		"libertyrepocache.zip",
+		"run-dev",
+		"run-debug",
+		"manifest.yml",
+		"idt.js",
+		".bluemix",
+		".build-ubuntu",
+		".yo-rc.json"}
+
+	// List of directories to not send to PFE
+	ignoredDirectories := []string{
+		".project",
+		"node_modules*",
+		".git*",
+		"load-test*",
+		".settings",
+		"Dockerfile-tools",
+		"target",
+		"mc-target",
+		".m2",
+		"debian",
+		".bluemix",
+		"terraform",
+		".build-ubuntu"}
+
+	ignoredList := ignoredFiles
+	if isDir {
+		ignoredList = ignoredDirectories
+	}
+
+	fileInIgnored := false
```

Comment:
Can this variable be called something more explicit to make it easier to read? fileInIgnored..?

Reply:
Changed to "isFileInIgnoredList"